src/paho/mqtt/client.py

```diff
@@ -24,8 +24,14 @@
 HAVE_SSL = True
 try:
```

Comment:
There's too much code in this try-catch. Everything after `import ssl` belongs in an `else` clause.